iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm

```diff
@@ -138,6 +139,9 @@ - (IBAction)login
       catch (exception const & ex)
       {
         LOG(LWARNING, ("Error login", ex.what()));
+        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
```

Comment:
С чего это он один? Авторизация кидает на форму, где вводишь логин-пароль осм, регистрация - на сайт осма, где проходит регистрация.
> On Mar 25, 2016, at 7:55 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm <https://github.com/mapsme/omim/pull/2511#discussion_r57463121>:
> 
> > @@ -138,6 +139,9 @@ - (IBAction)login
> >        catch (exception const & ex)
> >        {
> >          LOG(LWARNING, ("Error login", ex.what()));
> > +        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> Представь, что есть две кнопки: 1. Регистрация. 2. Авторизация. А по клику они вызывают один и тот же код. Вот именно это у нас и происходит.
> … <x-msg://53/#>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> 
>  <https://github.com/mapsme/omim/pull/2511#discussion_r57462735> <https://github.com/mapsme/omim/pull/2511#discussion_r57462415> <https://github.com/mapsme/omim/pull/2511#discussion_r57461965> <https://github.com/mapsme/omim/pull/2511#discussion_r57461714> <https://github.com/mapsme/omim/pull/2511#discussion_r57461616> <https://github.com/mapsme/omim/pull/2511#discussion_r57457291> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57457291> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461616> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461714> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461965> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57462415> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57462735>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57463121>